### PR TITLE
fix go to bottom binding for reflog

### PR DESCRIPTION
**Pull request by fsmiamoto**
I was able to reproduce the issue described at #1584 and adding an extra binding seemed to do the trick.

Let me know if the proper fix should be something else.

### CODE REVIEW

**Comment by fsmiamoto** [comment]
The `pushTag` test is still failing so my fix done at #1469 probably wasn't enough :eyes:

File: pkg/gui/list_context_config.go [hunk 1]
```diff
@@ -401,6 +401,7 @@ func (gui *Gui) getListContextKeyBindings() []*Binding {
 			{ViewName: listContext.GetViewName(), Tag: "navigation", Contexts: []string{string(listContext.GetKey())}, Key: gui.getKey(keybindingConfig.Universal.PrevPage), Modifier: gocui.ModNone, Handler: listContext.handlePrevPage, Description: gui.Tr.LcPrevPage},
 			{ViewName: listContext.GetViewName(), Tag: "navigation", Contexts: []string{string(listContext.GetKey())}, Key: gui.getKey(keybindingConfig.Universal.NextPage), Modifier: gocui.ModNone, Handler: listContext.handleNextPage, Description: gui.Tr.LcNextPage},
 			{ViewName: listContext.GetViewName(), Tag: "navigation", Contexts: []string{string(listContext.GetKey())}, Key: gui.getKey(keybindingConfig.Universal.GotoTop), Modifier: gocui.ModNone, Handler: listContext.handleGotoTop, Description: gui.Tr.LcGotoTop},
+			{ViewName: listContext.GetViewName(), Tag: "navigation", Contexts: []string{string(listContext.GetKey())}, Key: gui.getKey(keybindingConfig.Universal.GotoBottom), Modifier: gocui.ModNone, Handler: listContext.handleGotoBottom, Description: gui.Tr.LcGotoBottom},
```
Comment (jesseduffield) [inline pkg/gui/list_context_config.go:404]: if you look around line 413 we're setting the GotoBottom handler dynamically i.e. every context has a default handler for it but we override it for the commits view. Perhaps we should be only doing that override for the branch commits context given that the view will contain the reflog commits context as well

**Review by jesseduffield — CHANGES_REQUESTED** [review]
one thing

File: pkg/gui/list_context_config.go (hunk 1 above)
Comment (fsmiamoto) [reply pkg/gui/list_context_config.go]: Hmm now it makes sense, I've changed it to use the `Key` field to only apply the custom handler to the Branch Commits context

**Comment by jesseduffield** [comment]
nice work!

**Pull request merged**